PSOperations/PushCapability-iOS.swift

```diff
@@ -22,6 +22,12 @@ public struct Push: CapabilityType {
 
     public static let name = "Push"
     
+    public init(application: UIApplication) {
+        if authorizer.application == nil {
```

Comment:
Fewer lines of code, and checking for nil only makes sense because UIApplication is a singleton. 

No harm either way since it is a singleton, to be sure.